### PR TITLE
Dark spirit now only lets you move 21 tiles away

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

Dark spirit is one of the most OP items in the game because it gives you information, which is key to winning games. If you had perfect information every game, you'd win every game. Let's nerf it now before people realise how powerful information is.
# Spriting

# Wiki Documentation

Dark spirit now only lets you move 21 tiles away
# Changelog

:cl:  
tweak: Dark spirit now only lets you move 21 tiles away
/:cl:

### CODE REVIEW

**Comment by Mantlecrawler** [comment]
So this just makes it basically incredibly useless except for niche cases like spotted people outside a base or just hearing deadchat

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]
I'd understand like, 20-25 tiles, but 7 feels too small

**Comment by UltraBigRat** [comment]
Literally makes it worthless you might aswell just grab a zealots blindfold at this point and look at people with thermals

**Comment by JamieD1** [comment]
Too much of a nerf, dial it back

**Comment by Simplehorror** [comment]
@jamied1 done to 21, keep in mind you can browse other z-levels since +1 z only acts as 1 more in get_dist iirc

File: code/modules/antagonists/cult/runes.dm [hunk 1]
```diff
@@ -980,6 +980,9 @@ structure_check() searches for nearby cultist structures required for the invoca
 			if(affecting.health <= 10)
 				to_chat(G, span_cultitalic("Your body can no longer sustain the connection!"))
 				break
+
```
Comment (JamieD1) [inline code/modules/antagonists/cult/runes.dm]:
```suggestion
```

File: code/modules/antagonists/cult/runes.dm [hunk 2]
```diff
@@ -980,6 +980,8 @@ structure_check() searches for nearby cultist structures required for the invoca
 			if(affecting.health <= 10)
 				to_chat(G, span_cultitalic("Your body can no longer sustain the connection!"))
 				break
+			if (get_dist(G, affecting) > 21)
```
Comment (JamieD1) [inline code/modules/antagonists/cult/runes.dm]:
```suggestion
			if(get_dist(G, affecting) > 21)
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**